.ci/testruns/default/templates/testrun.yaml

```diff
@@ -44,3 +44,10 @@ spec:
       - name: REGION
         type: env
         value: eu-west-1
+  - name: dnsrecord-test
+    definition:
+      name: dnsrecord-test
+      config:
+      - name: USE_EXISTING_CLUSTER
```

Comment:
Should this be moved to `.test-defs/dnsrecord-test.yaml`?

Reply:
I am not really sure if it's even possible to specify an env variable in a `TestDefinition`. Even if it is, in g/g or extension repos this is right now never done. Instead, `TestDefinitions` only specify arguments but actual env variables are always specified in `Testruns`. Why should we deviate from this model here?

Reply:
My thinking behind the question was, that the `--kubeconfig` flag is also configured in the `TestDefinition`, which is basically where we want to say: "hey test, please use this existing cluster". So FMPOV, this is configuration that belongs together.
Thinking about it now, I guess both testrun and testdef are not used independently. So probably there's currently no value in separating cleanly here.